roles/oneagent/tests/conmponent/scripts/util/test_data_types.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import dataclass
+from enum import Enum
+from pathlib import Path
+from typing import Dict, List
+
+
+@dataclass
+class CommandResult:
+    returncode: int
+    stdout: str
+    stderr: str
+
+
+class DeploymentPlatform(Enum):
+    AIX_PPC = "aix_ppc"
```

Comment:
refactor the code to remove unnecessary abstraction

Reply:
Not sure what you mean here?